docs/docs-content/clusters/edge/trusted-boot/trusted-boot.md

```diff
@@ -37,6 +40,59 @@ software verified through cryptographic signatures. In the event that an Edge de
 release the key to decrypt the disk encryption if the boot process is tampered with, ensuring your user data remains
 encrypted.
 
+## Next Steps
+
+To get started with Trusted Boot, we recommend you start by familiarizing yourself with the concepts related to Trusted
+Boot, especially the keys used by Trusted Boot and how to manage them. In addition, Trusted Boot has additional hardware
+requirements compared with Edge hosts that do not have workloads with Trusted Boot. You should ensure that you use Edge
+hosts that meet the minimum hardware requirements.
+
+- [Trusted Boot Keys](./keys/keys.md)
+- [Key Management](./keys/key-management.md)
+- [Hardware Requirements](../hardware-requirements.md#trusted-boot)
+
+After understanding the core concepts, you can proceed to generate the keys to be used by your Edge host. You need to
+start by exporting the existing keys on your Edge host and then use those exported keys to generate new keys to be used
+by Trusted Boot. You can generate keys using a self-signed certificate, or an existing Certificate Authority (CA).
+
+- [Export Factory Keys](./keys/export-keys.md)
+- [Generate Trusted Boot Keys](./keys/generate-keys.md)
+
+With the keys ready, you can proceed to build the necessary Edge artifacts to install Palette on your Edge host and
+provision your cluster. The EdgeForge process with Trusted Boot is similar to the EdgeForge process without Trusted
+Boot. We recommend you familiar with the EdgeForge workflow first before building Edge artifacts with Trusted Boot
```

Comment:
```suggestion
Boot. We recommend you become familiar with the EdgeForge workflow first before building Edge artifacts with Trusted Boot
```

Reply:
Lenny if you want to call attention to this, you could even use an infobox if you want. Your call sir.